napari/_qt/__init__.py

```diff
@@ -1,6 +1,46 @@
-from ..resources import import_resources
+import os
+from distutils.version import StrictVersion
+from pathlib import Path
+from warnings import warn
+
+try:
```

Comment:
Does moving this here have any unexpected consequences for out "public" qt folder? I guess not as first thing there is `from .._qt.qt_main_window import Window` which will trigger this code, but I just want to check

Reply:
yeah, don't think so